src/ncar-bufr2nc-fortran/kinds.f90

```diff
@@ -36,38 +38,43 @@ module kinds
 !   machine:  ibm RS/6000 SP
 !
 !$$$ end documentation block
+!  use intrinsic fortran types
+  use,intrinsic :: iso_fortran_env
   implicit none
   private
 
 ! Integer type definitions below
 
 ! Integer types
-  integer, parameter, public  :: i_byte  = selected_int_kind(1)      ! byte  integer
-  integer, parameter, public  :: i_short = selected_int_kind(4)      ! short integer
-  integer, parameter, public  :: i_long  = selected_int_kind(8)      ! long  integer
-  integer, parameter, private :: llong_t = selected_int_kind(16)     ! llong integer
-  integer, parameter, public  :: i_llong = max( llong_t, i_long )
+  integer, parameter, public  :: i_byte  = int8       ! one byte integer
+  integer, parameter, public  :: i_short = int16      ! short integer
+  integer, parameter, public  :: i_long  = int64      ! long  integer = 64 bit on LP64 systems
+  integer, parameter, public  :: i_init = int32       ! 32 bit integer
```

Comment:
Shouldn't be `i_int`?

Reply:
@fcvdb Yes it has been changed now. Thank you for the catch.